__tests__/bundle/async3/simple.yml

```diff
@@ -0,0 +1,22 @@
+asyncapi: 3.0.0
+info:
+  title: Account Service
+  version: 1.0.0
+  description: This service is in charge of processing user signups
+channels:
+  userSignedup:
+    address: user/signedup
+    messages:
+      UserSignedUp:
+        $ref: '#/components/messages/UserSignedUp'
+operations:
+  sendUserSignedup:
+    action: send
+    channel:
+      $ref: '#/channels/userSignedup'
```

Comment:
Out of curiosity, is it even legal to reference through `channels`? I thought it was supposed to use the `components` section for shared components.

Reply:
No strict rules about that. We found this `simple.yaml` from async API examples inside their repos, so it must work.